packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -82,6 +82,18 @@ const trace = makeTracer('VD', true);
 
 const shortfallInvitationKey = 'shortfallInvitation';
 
+// exported for testing
+export const makeAllManagersDo = (collateralManagers, vaultManagers) => {
```

Comment:
swallowing errors like this seems unusual enough to merit a comment

Reply:
done. I also added a red flashing light 🚨

Reply:
Should this be yellow? we seem to be recovering from it.

But I guess it does indicate a bug - a "this should never happen" situation, not just some client mis-behaving or something. So ok, yes, red.

Reply:
> Should this be yellow? we seem to be recovering from it.

We recover from the loop, but we don't do anything about the individual vaultManager that died while setting up liquidations or locking prices. I think that's mostly okay because it will get another opportunity the next time around, but if one manager keeps failing, it should get attention.